src/Middleware/CORS/samples/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Tests do not work on Helix or when bin/ directory is not in project directory due to undeclared dependency on test content. -->
```

Comment:
Suggest correcting this comment to comment on the _sample_.

Reply:
This is one of the cases where the folder probably should have been named "testassets" instead of samples. The tests depend on these samples.